src/Features/LanguageServer/Protocol/Microsoft.CodeAnalysis.LanguageServer.Protocol.csproj

```diff
@@ -19,7 +19,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="$(MicrosoftExtensionsDependencyInjectionVersion)" />
```

Comment:
You probably want to remove the Version entry as well since I added it for this: https://github.com/dotnet/roslyn/commit/f6a8179cc2791d68c66e35637740e671caaac820.

Reply:
it should be still used in the example project - https://github.com/dibarbet/roslyn/blob/remove_di/src/Features/LanguageServer/Microsoft.CommonLanguageServerProtocol.Framework.Example/Microsoft.CommonLanguageServerProtocol.Framework.Example.csproj#L12